grpc/src/main/java/io/stargate/grpc/impl/GrpcImpl.java

```diff
@@ -28,13 +29,20 @@
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
 import java.net.UnknownHostException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class GrpcImpl {
   private static final Logger logger = LoggerFactory.getLogger(GrpcImpl.class);
+  private static final Integer EXECUTOR_SIZE =
+      Integer.getInteger(
+          "stargate.grpc.executor_size", Runtime.getRuntime().availableProcessors() * 2);
```

Comment:
will this `Runtime.getRuntime().availableProcessors()` result in a sane number in the shared k8s environment?

Reply:
Looks like it works in everything after 8u131
https://blogs.oracle.com/java-platform-group/java-se-support-for-docker-cpu-and-memory-limits

Reply:
Wait, actually might not apply until JDK 10
https://dev.to/zac_siegel/java-and-docker---memory-and-cpu-limits-3h77

Reply:
Good catch. If we can't trust that method, then the only alternative is a hard-coded number.